backend/api/nomisUsersAndRolesApi.js

```diff
@@ -18,9 +21,36 @@ const nomisUsersAndRolesFactory = (client) => {
     )
   const getCaseloads = (context) => get(context, '/reference-data/caseloads')
 
+  const removeRole = (context, username, roleCode) => del(context, `/users/${username}/roles/${roleCode}`)
+  const addRole = (context, username, roleCode) => put(context, `/users/${username}/roles/${roleCode}`)
+  const addUserRoles = (context, username, roles) => post(context, `/users/${username}/roles`, roles)
+  const getUser = (context, username) => get(context, `/users/${username}`)
+  const getUserCaseloads = (context, username) => get(context, `/users/${username}/caseloads`)
+  const userCaseLoads = (context, username) =>
+    context.authSource !== 'auth' ? getUserCaseloads(context, username) : []
+  const getRoles = (context, hasAdminRole) => get(context, `/roles?admin-roles=${hasAdminRole}`)
+  const contextUserRoles = (context, username) => get(context, `/users/${username}/roles`)
```

Comment:
I think it would have been better if this was simple called `getRolesForUser` but I why you have kept to old names